### ISSUE TITLE
Support Infobox image params

### ISSUE THREAD

**Issue by yanosh-igor** [spencermountain/wtf_wikipedia#165]
Hi, I have noticed wrong image filenames.
For instance, if you parse this article https://en.wikipedia.org/wiki/David_Koresh
and get infobox images then the parser will return ` {file: "David koresh.jpgborder", text: ""}` but should be ` {file: "David koresh.jpg", text: ""}`
It adds word `border` to the end of file name.
Here is snippet from dump:
```
{{Infobox person\n| name        = David Koresh
| image       = David koresh.jpg{{!}}border
| image_size  =
| caption     = Koresh in 1987
```
![2018-09-08_1542](https://user-images.githubusercontent.com/435827/45254243-cf007e00-b37d-11e8-9b6e-055b678d2d5d.png)
Possibly there could be other words after image filename. Will be good to ignore them somehow.

**Comment by spencermountain** [comment]
hah!
[what the?!](https://www.mediawiki.org/wiki/Help:Magic_words#Other)
i guess infobox image supports some piped params, and that's a tricky-pipe

oof!

**Comment by spencermountain** [comment]
this example is fixed too in `5.3.0`
